doc/xsd/tester_config.xsd

```diff
@@ -194,4 +203,21 @@
         </xsd:attribute>
     </xsd:complexType>
 
+    <xsd:complexType name="SyntaxOptions">
+        <xsd:annotation>
+            <xsd:documentation>
+                Type of element to specify options on how to handle xml syntax.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:choice minOccurs="0" maxOccurs="unbounded">
+            <xsd:element name="strip_indent" type="xsd:string">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Enable or disable strip indentation.
```

Comment:
Enable indentation stripping.

Disabled by default.

Also it does not say where does it strip indentation.